point_pattern.py

```diff
@@ -33,6 +33,8 @@ def read_geojson(input_file):
     """
     # Please use the python json module (imported above)
     # to solve this one.
+    with open(input_file, 'r') as f:
+        gj = json.load(f)
```

Comment:
Go ahead and print the input_file here to see if you are passing the PATH in correctly.  What does input_file equal?